### PR TITLE
yunyun/request.js

### PR DESCRIPTION
**Pull request by yunyun2010**
一个用于请求API的扩展。其实是我自己想用(

### CODE REVIEW

**Comment by xiaochen004hao** [comment]
有点酷

File: yunyun/request.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                defaultValue: "https://www.example.com/",
+              },
+            },
+          },
+        ],
+      };
+    }
+
+    get(args) {
+      return Scratch.fetch(args.URL)
```
Comment (FurryR) [inline yunyun/request.js:31]: Gandi 不允许外链请求，所有使用此功能的扩展均不能通过审核。

**Review by FurryR — CHANGES_REQUESTED** [review]
已审核，未通过。

File: yunyun/request.js (hunk 1 above)
Comment (fath11) [reply yunyun/request.js:31]: POV Scratch Hub: 🤣🤣

Comment (xiaochen004hao) [reply yunyun/request.js:31]: 可怜

**Comment by qxsck** [comment]
你说得对，但是fetch拓展过不了审，而且你这怕不是tw那copy来的吧，/* generated l10n code */都没删

**Comment by MoreBugOfDog** [comment]
Hello! The developer is cloudy!
Since Gandi-IDE and cocrea don't allow extensions to request external links (a security risk), it's unlikely that your extension will pass the review! I suggest you close this PR to avoid adding pointless work for reviewers.
Next time you make an extension it is recommended to know the ground rules and read the PRs of those who have gone before you (because there are a lot of extensions that have not passed the review because of the ban on outbound links)
(I may not sound very nice, please forgive me)

**Comment by MoreBugOfDog** [comment]
Also, before moving or adapting extensions from other scratch communities, please get authorization from the original authors, review the content of the extensions, and test that the extensions work correctly in the Gandi-IDE (you're moving the extensions directly using Scratch.extensions.register, not Gandi-IDE's required window.tempExt, which can't be loaded using Gandi-IDE's official loader).

**Comment by DilemmaGX** [comment]
搬运扩展不是不可以，但需要征求原作者意见，并且明确双方贡献，详细地记录下来，tempExt里扩展所有者应当是原作者。
除此之外，这个扩展貌似不能被Gandi正常加载（而且跨域似乎是不被允许的）。

**Comment by MoreBugOfDog** [comment]
⚠️ must be noted: if an extension from someone else is ported or adapted, developer information must be included within the metadata! This is basic ethics and is not affected by the license.

**Comment by MoreBugOfDog** [comment]
Moving extensions is not out of the question, but the original author needs to be consulted and contributions from both sides need to be clear and documented in detail, and the extension owner should be the original author in tempExt.
Other than that, the extension doesn't seem to be loaded properly by Gandi (and cross-domain seems to be disallowed).
[DilemmaGX](https://github.com/DilemmaGX) Reviews in English

**Comment by MoreBugOfDog** [comment]
Did you even just change the extension ID in the comments?

**Comment by MoreBugOfDog** [comment]
Suggest to close this PR to recreate another extension

**Comment by MoreBugOfDog** [comment]
https://extensions.turbowarp.org/fetch.js
The same extension as yours exists in turbowarp.
Unconsented removal and unattributed behavior is true.

**Comment by xiaochen004hao** [comment]
### **_`吃瓜ing`_**

**Comment by yunyun2010** [comment]
我想要API

**Comment by qxsck** [comment]
> 我想要API

你需要这些api有何用？继续你的搞破坏吗

**Comment by MoreBugOfDog** [comment]
> > 我想要API
> 
> 你需要这些api有何用？继续你的搞破坏吗

同问

**Comment by kukemc** [comment]
无意义的pr，槽点过多

**Comment by six-ddc** [comment]
Sorry, due to the lack of a reliable policy to ensure the security of network requests, Gandi does not currently support the related extensions.
抱歉，由于缺乏可靠的策略来确保网络请求的安全，Gandi目前不支持相关扩展。

**Pull request closed without merging**